XamarinCommunityToolkit/Layouts/HexLayout.shared.cs

```diff
@@ -0,0 +1,278 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Layouts
```

Comment:
There is a test failing over this one. I see we technically have in the README that Layouts is another namespace (but then .UI.Layouts) do we want to actually have that or put it under Views?

Either way is fine with me but then we need to update the test :)

Reply:
I'd prefer to put it inside Views

Actually, Expander and SideMenu are a layouts too

Reply:
Ok, I am going to move it to Views namespace (and folder).

Reply:
Moved.